src/pages/documentation/adobe-analytics/migrate-to-edge-network.md

```diff
@@ -83,3 +83,25 @@ When using Identity for Edge Network extension, the Mobile SDK automatically mig
 The previous Identity for Experience Cloud ID Service extension can be safely removed from the SDK registration process if there are no other extensions which rely on it, such as Target, Campaign and others.
 
 If you need further assistance, please contact [Adobe Experience Cloud customer care](https://experienceleague.adobe.com/?support-solution=General#support).
+
+## Locations of the Extensions
```

Comment:
@cacheung wondering if this section is required since we already link all repos in the current sdk versions page, so instead can we point to that page:

## Next steps

To learn more about latest versions available for the Edge Network and Edge Bridge extensions, see the Current SDK versions page (link)